### DISCUSSION TITLE
Bridgetown + Lookbook

### DISCUSSION THREAD

**Discussion by work-leonid** [bridgetownrb/bridgetown#587] [Q&A] [answered]
Hi! I sometimes use bridgetown for quick prototypes. The only thing I miss right now is a nice preview of the components.

[Lookbook](https://github.com/allmarkedup/lookbook) is a great native Ruby & ViewComponents way to show components. Has anyone tried bridgetown & lookbook? I did but it didn't work. Maybe someone can share some direction

**Comment by KonnorRogers** [comment, marked as answer]
I can't imagine it would work with Bridgetown considering it's not native Ruby and is heavily integrated with Rails.

https://github.com/allmarkedup/lookbook/blob/main/lib/lookbook/engine.rb

**Reply by work-leonid** [reply]
Got it. I thought it was more "native" and simple. That's too bad. 

Thanks for the reply @ParamagicDev!

**Reply by jaredcwhite** [reply]
@work-leonid An actual Storybook <=> Bridgetown integration is something I've been prototyping. Storybook isn't so great in certain ways, _but_ it is very well understood in the industry and it can handle purely frontend components in addition to Ruby-backed HTML, so I'm kinda hoping to make it Just. Work. ™️

**Reply by work-leonid** [reply]
Great! Thank you for your work.  

Sometimes I want to be able to quickly show a few components and let people copy the code, see how components behave in different sizes in a user-friendly interface. Storybook and Lookbook are perfect for that.
